pages/resources/jobExplorer/productManager/index.tsx

```diff
@@ -1,7 +1,6 @@
 import React from "react";
-import LandingNavbar from "../../../components/landingPage/LandingNavbar";
-import ProductManagerComponent from "../../../components/resources/jobExplorer/productManager/ProductManagerComponent";
-
+import LandingNavbar from "../../../../components/landingPage/LandingNavbar";
+import ProductManagerComponent from "../../../../components/resources/jobExplorer/productManager/ProductManagerComponent";
 export default function ProductManagerPage() {
```

Comment:
@Curtoista  I used `index.tsx` files inside the folder name `productManager`

We should align on how you architect this page, because I think I'm doing it differently to you.

Reply:
reverted, as per our OH convo yesterday